tools/plugins/com.liferay.ide.gradle.ui/src/com/liferay/ide/gradle/ui/LiferayGradleUI.java

```diff
@@ -39,10 +39,14 @@
  * @author Gregory Amerson
  * @author Simon Jiang
  */
-public class GradleUI extends Plugin {
+public class LiferayGradleUI extends Plugin {
 
 	// The plug-in ID
 
+	public static final String LIFERAY_STANDALONE_WATCH_JOB_SUFFIX = "standalone";
```

Comment:
The user should not see the word 'standalone' or 'workspace' at the end of the job names.  They don't understand what those means.  @simonjhy  please send another PR (doesn't have to be for 3.5 release, but by 3.6) where we don't use the 'display' name of a job to determine what type of job it is.  IF we need to differentiate between the two, we should do it using jobFamlies not job names.